### DISCUSSION TITLE
Installing error on termux

### DISCUSSION THREAD

**Discussion by eelalzep** [ocrmypdf/OCRmyPDF#1429] [Q&A] [answered]
I installed every dependencies, but still getting errors after errors.
![Screenshot_2024-11-10-23-42-11-375_com.termux.jpg](https://github.com/user-attachments/assets/5e396ed4-99bf-4f8c-9271-889c7534ae3d)

**Comment by jbarlow83** [comment, marked as answer]
Thank you for providing further details.

It looks like you reported the actual issue here so I will close
https://github.com/bigcat88/pillow_heif/issues/310
